frontend/src/tests/components/PersonalSchedules/PersonalScheduleForm.test.js

```diff
@@ -0,0 +1,126 @@
+import { render, waitFor, fireEvent } from "@testing-library/react";
+import PersonalScheduleForm from "main/components/PersonalSchedules/PersonalScheduleForm";
+import { personalSchedulesFixtures } from "fixtures/personalSchedulesFixtures";
+import { BrowserRouter as Router } from "react-router-dom";
+
+const mockedNavigate = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useNavigate: () => mockedNavigate
+}));
+
+
+describe("PersonalScheduleForm tests", () => {
+
+    test("renders correctly ", async () => {
+
+        const { getByText } = render(
+            <Router  >
+                <PersonalScheduleForm />
+            </Router>
+        );
+        await waitFor(() => expect(getByText(/Name/)).toBeInTheDocument());
+        await waitFor(() => expect(getByText(/Create/)).toBeInTheDocument());
+    });
+
+
+    test("renders correctly when passing in a PersonalSchedule ", async () => {
+
+        const { getByText, getByTestId } = render(
+            <Router  >
+                <PersonalScheduleForm initialPersonalSchedule={personalSchedulesFixtures.onePersonalSchedule} />
+            </Router>
+        );
+        await waitFor(() => expect(getByTestId(/PersonalScheduleForm-id/)).toBeInTheDocument());
+        expect(getByText(/Id/)).toBeInTheDocument();
+        expect(getByTestId(/PersonalScheduleForm-id/)).toHaveValue("1");
+    });
+
+
+    /*test("Correct Error messsages on bad input", async () => {
+
+        const { getByTestId, getByText } = render(
+            <Router  >
+                <PersonalScheduleForm />
+            </Router>
+        );
+        await waitFor(() => expect(getByTestId("PersonalScheduleForm-name")).toBeInTheDocument());
+        // const nameField = getByTestId("PersonalScheduleForm-name");
+        // const descriptionField = getByTestId("PersonalScheduleForm-description");
+        const submitButton = getByTestId("PersonalScheduleForm-submit");
+
+        // fireEvent.change(quarterYYYYQField, { target: { value: 'bad-input' } });
+        // fireEvent.change(localDateTimeField, { target: { value: 'bad-input' } });
+        fireEvent.click(submitButton);
+
+        await waitFor(() => expect(getByText(/Description is required./)).toBeInTheDocument());
+        expect(getByText(/Name is required./)).toBeInTheDocument();
+    });*/
```

Comment:
Unless necessary, I would delete it because Bryan Terce will probably make you delete it anyway